File: snippets/php/send_email_from_a_subaccount.php
require_once('/path/to/mailchimp-transactional-php/vendor/autoload.php');
function run($message)
{
    try {
        $mailchimp = new MailchimpTransactional\ApiClient();
        $mailchimp->setApiKey("YOUR_API_KEY");
        $response = $mailchimp->messages->send(["message" => $message]);
        print_r($response);
    } catch (Error $e) {
        echo 'Error: ', $e->getMessage(), "\n";
    }
}

$message = [
    "from_email" => "hello@example.com",
    "subject" => "Subaccount test",
    "text" => "This email was sent from a subaccount!",
    "to" => [
        [
            "email"=>"freddie@example.com",
            "type"=>"to"
        ]
    ],
    "subaccount"=>"UNIQUE_ACCOUNT_ID_PHP"
];
run($message);

File: snippets/ruby/send_email_from_a_subaccount.rb
require 'MailchimpTransactional'

client = MailchimpTransactional::Client.new(ENV['YOUR_API_KEY'])

message = {
  from_email: "hello@example.com",
  subject: "Subaccount test",
  text: "This email was sent from a subaccount!",
  to: [
    {
      email: "freddie@example.com",
      type: "to"
    }
  ],
  subaccount: "UNIQUE_ACCOUNT_ID"
}

begin
  msg_response = client.messages.send(message: message)
  puts msg_response
rescue MailchimpTransactional::ApiError => e
  puts "Error: #{e}"
end